tests/unit/Framework/MockObject/MockObjectTest.php

```diff
@@ -27,6 +28,7 @@
 #[Group('test-doubles/mock-object')]
 #[TestDox('Mock Object')]
 #[Medium]
+#[CoversClass(MockObject::class)]
```

Comment:
`MockObject` is an interface and cannot/should not be targeted for coverage.

Reply:
yeah, I also realized it.. I am in the process of fixing the rule in https://github.com/rectorphp/rector-phpunit/pull/325